examples/remote_syslog.ebextensions.config

```
@@ -34,7 +34,8 @@ files:
       #!/bin/bash
       logger_config="/etc/log_files.yml"
       appname=`{ "Ref" : "AWSEBEnvironmentName" }`
-      instid=`wget -q -O - http://169.254.169.254/latest/meta-data/instance-id`
+      IMDS_TOKEN=`curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600"`
+      instid=`curl -H "X-aws-ec2-metadata-token: $IMDS_TOKEN" http://169.254.169.254/latest/meta-data/instance-id`
```

Couple minor suggestions for consistency:
* Let's use `curl -s` as it's equivalent to the original `wget -q`
* Keep the URL last, and it doesn't need quotes.

```suggestion
      IMDS_TOKEN=`curl -s -X PUT -H "X-aws-ec2-metadata-token-ttl-seconds: 21600" http://169.254.169.254/latest/api/token`
      instid=`curl -s -H "X-aws-ec2-metadata-token: $IMDS_TOKEN" http://169.254.169.254/latest/meta-data/instance-id`
```

Does that look OK?

---

@markdascher Looks good! Good catch with the `-s` option to keep it consistent with original code. I pulled the first curl from the AWS docs, but this change is good.